test/e2e/storage/testsuites/disruptive.go

```diff
@@ -47,11 +47,14 @@ func InitDisruptiveTestSuite() TestSuite {
 		},
```

Comment:
@mkimuram @jsafrane what do you think of removing the disruptive tests for the FS volume mode?  It should already be covered by subpath disruptive tests.

Reply:
Pushed new change removing them

Reply:
In my understanding, (D-2) is subset of (S-1) and (D-3) is subset of (S-2). Differences between them are whether they check subpath or not.
So, (D-2) and (D-3) will be covered by subpath tests, but (D-1) won't.
To test (D-1), it would be worth keeping disruptive tests for filesystem.
(Instead, we will be able to modify `runTestFile` in `disruptiveTestTable` to skip (D-2) and (D-3), if needed.)

```
[Subpath test]
  (S-1) "should unmount if pod is gracefully deleted while kubelet is down [Disruptive][Slow][LinuxOnly]"
  (S-2) "should unmount if pod is force deleted while kubelet is down [Disruptive][Slow][LinuxOnly]"

[Disruptive test]
  (D-1) "Should test that pv written before kubelet restart is readable after restart."
  (D-2) "Should test that pv used in a pod that is deleted while the kubelet is down cleans up when the kubelet returns."
  (D-3) "Should test that pv used in a pod that is force deleted while the kubelet is down cleans up when the kubelet returns."
```

Reply:
Good catch, pushed a new commit that disables the last 2 tests for file mode.